RandomMonomialIdeals.m2

```diff
@@ -112,20 +112,24 @@ doc ///
   randomGeneratingSets
   (randomGeneratingSets,ZZ,ZZ,RR,ZZ)
  Headline
-  randomly generates lists of monomials, up to a given dimension
+  randomly generates lists of monomials, up to a given degree
  Usage
   randomGeneratingSets (ZZ,ZZ,RR,ZZ)
  Inputs
-  n: ZZ
-  D: ZZ
-  p: RR
-  N: ZZ
+  n:ZZ
+    number of variables
+  D:ZZ
+    maximum degree
+  p:RR
+    probabilty to select a monomial
+  N:ZZ
+    number of sets generated
  Outputs
   B: List
-   a list of generating sets of monomials
+   random generating sets of monomials
  Description
   Text
-   given number of variables n, degree bound D, and probability parameter p, N generating sets of monomials are randomly generated
+   randomMonomialIdeals first generates all possible monomials with a degree 1 to D and number of variables 1 to n. Once all possible monomials are generated, each monomial has a probabilty p to be selected. The monomials selected are then outputted as a random set. This function does not include the monomial with degree 0.
```

Comment:
also not a critical comment, but just a suggestion:
I think something like "probability of selecting each monomial" is slightly clearer than "probability to select a monomial" for the description of the input p